### ISSUE TITLE
Jest warning "`new NativeEventEmitter()` was called with a non-null argument"

### ISSUE THREAD

**Issue by Andarius** [rnmapbox/maps#1660]
**Describe the bug**  

When running jest tests, the following warning is raised

```
console.warn
    `new NativeEventEmitter()` was called with a non-null argument without the required `addListener` method.

      at new NativeEventEmitter (node_modules/react-native/Libraries/EventEmitter/NativeEventEmitter.js:61:17)
      at Object.<anonymous> (node_modules/@react-native-mapbox-gl/maps/javascript/modules/location/locationManager.js:6:43)
      at Object.<anonymous> (node_modules/@react-native-mapbox-gl/maps/javascript/components/UserLocation.js:4:1)
```

**To Reproduce**  

Run `jest` on a component using `<MapboxGL.MapView />`

**Expected behavior**  
Not to raise a warning

**Actual behavior**  
A warning is raised

**Versions (please complete the following information):**  
 - Platform: NA
 - Platform OS: NA
 - Device: NA
 - Emulator/ Simulator: NA
 - Dev OS: KDE Neon
 - react-native-mapbox-gl Version:  8.5.0
 - React Native Version : 0.66.3

**Comment by ferdicus** [comment]
hi, can you please post your jest setup/config as well - thanks

**Comment by Andarius** [comment]
FYI my current workaround is to add `jest.mock('react-native/Libraries/EventEmitter/NativeEventEmitter');` inside my mocks file.

**Comment by mfazekas** [comment]
Should be fixed by #1760
